### ISSUE TITLE
Storybook starts with error: Failed to execute setItem on Storage

### ISSUE THREAD

**Issue by Donaldini** [kadirahq/react-storybook#452]
On starting storybook I get the following error

`Failed to execute 'setItem' on 'Storage': Setting the value of 'cb0320b0-988a-47a9-8aa0-c7a963e524bb' exceeded the quota.`

Already tried...
  * I updated to the latest version (@kadira/storybook@2.13.1) to see if the problem was resolved but alas.
  * Removing the @kadira folder from node_modules and readding it with npm update

<img width="1436" alt="screen shot 2016-09-14 at 10 32 09" src="https://cloud.githubusercontent.com/assets/2185680/18505213/8b85b08e-7a66-11e6-8c35-16a17af635ac.png">

**Comment by mnmtanish** [comment]
Hi @Donaldini 
Can you share the repo ( or an example repo with the issue ).

**Comment by Donaldini** [comment]
Couldn't share the code because it's a private repo, but we managed to fix the issue ourselves (should've waited longer to post this :-) ) .

The problem was linked to the local storage of the browser (in my case Chrome 53.0.2785.113), because when I tried it in Firefox, which I don't often use, the problem was non-existent.

I can only show you what it currently looks like in Firefox (even though it's not giving me issues just yet): 

![image](https://cloud.githubusercontent.com/assets/2185680/18507267/efa90418-7a6f-11e6-9a71-36a8f09c16de.png)

My Chrome > Developer Tools > Application > Storage > Local Storage was filled with items like the screenshot above.

After clearing the storage the problem was resolved. Maybe Storybook isn't cleaning its sessions that good up until the point where the local storage is saturated? :unsure:

![sep -14-2016 11-43-20](https://cloud.githubusercontent.com/assets/2185680/18507392/7febc614-7a70-11e6-82a0-e0f2bf44e2a5.gif)
 
**TL;DR**
  - Storybook filled up Local Storage
  - Clearing Local Storage solved problem :-)

**Comment by arunoda** [comment]
We may need to look at how we use local storage.

**Comment by mnmtanish** [comment]
Fixed with `v2.20.0`
